docs(domains): added how-to renew domain (#1396)
scaleway/docs-content: network/domains-and-dns/how-to/renew-domain.mdx

---
meta:
  title: How to renew a domain name
  description: This page explains how to renew a domain name with Scaleway Domains and DNS.
content:
  h1: How to renew a domain name
  paragraph: This page explains how to renew a domain name with Scaleway Domains and DNS.
tags: renew domain dns
dates:
  validation: 2023-03-07
  posted: 2023-03-07
categories:
  - network
---

> This page shows you ho to renew your domain name.

Reviewer: ```suggestion
This page shows you how to renew your domain name.
```

<Macro id="iam-requirements" />
 
<Message type="requirement">
  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
  - You have entered your credit card information on your account.
  - Your have registered one or more domain names with Scaleway Domains and DNS.
</Message>

1. Click **Domains and DNS** in the **Network** section of the [Scaleway console](https://console.scaleway.com) side menu. The list of your domains displays.
2. Click the domain you want to renew. Your domain's overview page displays.
3. Click **Renew** under **Expiration date**. A pop-up displays.
4. Choose your registration period in the drop-down list.

    <Message type="note">

>      You can renew your domain for one to nine years with a maximum domain name registration period of 10 years.

Reviewer: ```suggestion
     You can renew your domain for up to nine years with a maximum domain name registration period of 10 years.
```
Pas sûre d'avoir compris, l'idée c'est qu'on peut renouveler son domaine un maximum de 9 ans ?

>      You can renew your domain for one to nine years with a maximum domain name registration period of 10 years.

Author: ```suggestion
     You can renew your domain for up to 9 years with a maximum domain name registration period of 10 years.
```

    </Message>

> 5. Check the box to confirm you understand your domain will be renewed and that you will be debited the price that displays.

Reviewer: ```suggestion
5. Check the box to confirm you understand your domain will be renewed and that you will be debited the displayed price.
```
